Quickstart uninformative links and Grammarly edits (#189)
hubverse-org/hubDocs: docs/source/quickstart-hub-admin/uploading-validating.md

# Uploading and validating config files  

You have now made all the changes necessary to start using the simple hub, but the changes to `admin.json` and `tasks.json` are only on your local computer. You should also apply them to your GitHub repository.

## Uploading changes from your local computer to GitHub  

Follow [GitHub directions](https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository) to add files to a repository.  

Once you have uploaded those two files onto your GitHub repository, you will notice that they have been updated by looking at the last commit date, as indicated below.  

```{image} ../images/github-commits.png
:alt: Screenshot of GitHub repository page with arrows pointing at the time of last commits
:class: bordered
```

## Validating your config files  

Now, you should validate the config files to be sure they are properly functional. You can use the `validate_config` function from [`hubAdmin`](https://hubverse-org.github.io/hubAdmin/index.html) to check whether Hub config files are valid. The steps are as follows:  

> 1. First, you must install the R package `hubAdmin`. Click on the link for [instructions for installing `hubAdmin` in the R console](https://github.com/hubverse-org/hubAdmin#installation).  

Reviewer: ```suggestion
1. First, you must [follow the instructions to install the R package `hubAdmin` in the R console](https://github.com/hubverse-org/hubAdmin#installation).  
```

2. Next, you need to validate the Hub's config files. 
- [`validate_hub_config()`](https://hubverse-org.github.io/hubAdmin/reference/validate_hub_config.html) validates the `admin.json`, `tasks.json`, `model-metadata-schema.json` hub config files in a single call.

> - [`validate_config()`](https://hubverse-org.github.io/hubAdmin/reference/validate_config.html) validates a single hub config file against its corresponding schema.

Reviewer: This was confusing to me when I started because `validate_hub_config()` actually calls `validate_config()`. I think it would be better to just leave it out. The documentation explains what's happening. 

```suggestion
```

- If you get any errors, you can pass the result of the above functions to `view_config_val_errors()`, which prints a concise and informative version of the validation errors table.

> - Click on the link for more [detailed instructions and explanations](https://hubverse-org.github.io/hubAdmin/articles/hub-setup.html#validate-config-files).

Reviewer: ```suggestion
- More detailed explanations can be found in [hubAdmin's "Setting Up a Hub" vignette](https://hubverse-org.github.io/hubAdmin/articles/hub-setup.html#validate-config-files).
```
